spec/lib/pdf_filler/nj1040_pdf_spec.rb

```diff
@@ -144,6 +144,13 @@
           expect(pdf_fields["undefined_9"]).to eq "0"
         end
 
+        it "fills pdf with Line 8 fields" do
```

Comment:
[dust] Maybe we should specify the scenario when we expect these fields to be present but not checked